### PR TITLE
added tenant in query param

### PR DESCRIPTION
**Pull request by tejassonar**
Removed tenant_key from headers and added in query params

### CODE REVIEW

**Review by sarveshpro — APPROVED** [review]
LGTM ✅

**Review by norbertschuler — COMMENTED** [review]
It is unusual to add query params to POST API calls, e.g. POST https://app-staging.plant-for-the-planet.org/app/donations?tenant=ten_KRHYP8TR or POST https://app-staging.plant-for-the-planet.org/app/donations/don_B0fQrhNzQr7inJOTxss0xQvh/publish?tenant=ten_KRHYP8TR - does the backend can cope with this?

File: src/Layout/QueryParamContext.tsx [hunk 1]
```diff
@@ -337,18 +341,18 @@ export default function QueryParamProvider({ children }: any) {
...
+  //     // TODO => verify tenant before setting it
+  //     settenant(router.query.tenant);
+  //     localStorage.setItem("tenant", router.query.tenant);
+  //   } else {
+  //     localStorage.removeItem("tenant");
+  //   }
+  //   return () => {
+  //     localStorage.removeItem("tenant");
+  //   };
+  // }, [router.query.tenant]);
```
Comment (norbertschuler) [inline src/Layout/QueryParamContext.tsx:355]: If we do not need this anymore, let's remove this.

File: src/Utils/api.ts [hunk 2]
```diff
@@ -30,15 +30,15 @@ axiosInstance.interceptors.request.use(
...
-    }
+    // if (typeof Storage !== "undefined") {
+    //   config.headers["tenant-key"] = `${
+    //     localStorage.getItem("tenant")
+    //       ? localStorage.getItem("tenant")
+    //       : "ten_I9TW3ncG"
+    //   }`;
+    // } else {
+    //   config.headers["tenant-key"] = "ten_I9TW3ncG";
+    // }
```
Comment (norbertschuler) [inline src/Utils/api.ts:41]: If we do not need this anymore, let's remove this.

**Review by norbertschuler — APPROVED** [review]
Seems to work for me.

File: src/Layout/QueryParamContext.tsx (hunk 1 above)
Comment (tejassonar) [reply src/Layout/QueryParamContext.tsx]: done

File: src/Utils/api.ts (hunk 2 above)
Comment (tejassonar) [reply src/Utils/api.ts]: done

Comment (jmiridis) [reply src/Utils/api.ts]: Backend can handle this.
The only issues that might arise can be related to caching but I think that's another subject,

**Pull request merged**